src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -2658,6 +2658,19 @@ bool primitive_inst::is_valid_fusion() const {
         if (fd.is_type<eltwise>())
             can_broadcast = ov::PartialShape::broadcast_merge_into(merged_shape, outer_dep_pshape, fd.typed_desc<eltwise>()->broadcast_spec);
 
+        // Check if broadcast happens more than single axis.
+        // Current FUSED_OP_LOAD macro cannot support broadcast on dynamic dimension.
+        if (can_broadcast == true && (merged_shape.is_static() && outer_dep_pshape.is_static()) &&
```

Comment:
Are these two conditions needed? At runtime, shapes should be static always.

Reply:
You are right, I removed it. Thanks!